images/operator-sdk/Dockerfile

```diff
@@ -1,7 +1,10 @@
 FROM registry.access.redhat.com/ubi8/ubi-minimal:latest
 
-COPY operator-sdk /usr/local/bin/operator-sdk
+ENV CGO_ENABLED=0
```

Comment:
What prompted the addition of setting CGO_ENABLED=0 for callers of the `operator-sdk` docker image? Is it required that operator projects are _also_ compiled with CGO_ENABLED=0?

Reply:
I don't think its necessary (and will remove it). I added it because I saw golang being installed in this image. On the flip side, do you expect this setting to propagate to the operator project in some way? I don't see how this would affect operator compilation.

Reply:
It would propagate to an operator build if someone ran `docker run --rm -t -v $(pwd):/app -w /app quay.io/operator-framework/operator-sdk make generate manifests bundle test manager` (the `test` and `manager` targets compile Go code)

I'm not sure how prevalent this type of thing would be, but if `go` is installed in the image, I think we can assume that somebody is going to use it to compile their operator.

Reply:
I guess people can do whatever they want within the image.